modules/swagger-codegen/src/main/resources/TypeScript-Fetch/api.mustache

```diff
@@ -93,12 +93,25 @@ export const {{classname}}FetchParamCreator = {
 
         let contentTypeHeader: Dictionary<string> = {};
 {{#hasFormParams}}
-        contentTypeHeader = { "Content-Type": "application/x-www-form-urlencoded" };
-        fetchOptions.body = querystring.stringify({
-        {{#formParams}}
-            "{{baseName}}": params["{{paramName}}"],
-        {{/formParams}}
-        });
+        let formData: Dictionary<any> = {};
+        let isMultipartData = false;
+{{#formParams}}
+        {{#isFile}}isMultipartData = true;
+        {{/isFile}}formData["{{baseName}}"] = params["{{paramName}}"];
+{{/formParams}}
+
+        if (isMultipartData) {
```

Comment:
Why not instead of this if statement in all the generated code. Use a Mustache if. This way you don't need the next strange code:
```
let isMultipartData = false;
isMultipartData = true;
```
maybe do something like this:
```
{{#isFile}}
  let body = new FormData();
  for (let key in formData) {
     body.append(key, formData[key]);
  }
 fetchOptions.body = body;
 contentTypeHeader = { "Content-Type": "multipart/form-data" };
{{/isFile}}
{{^isFile}}
   fetchOptions.body = querystring.stringify(formData);
   contentTypeHeader = { "Content-Type": "application/x-www-form-urlencoded" };
{{#isFile}}
```